Projects/Backendless.NET.Framework4.0/bin/Debug/Backendless.dll.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
why does Backendless.dll need a dependency to Newtonsoft.Json? We do not use that library anywhere. Socket IO does, but we do not

Reply:
@markpiller, tt is installed along with `SocketIoDotNetClient` and `EngineIoDotNetClient`. I cannot remove this package.